python/tvm/relay/frontend/pytorch.py

```diff
@@ -337,6 +337,55 @@ def _impl(inputs, input_types):
         return _op.transform.repeat(data, repeats=repeats, axis=axis)
     return _impl
 
+
+def _parse_input_data(inp):
+    import torch
+    if isinstance(inp, _expr.Var):
+        data = inp
+    elif isinstance(inp, _expr.Call):
+        data = inp
+    elif isinstance(inp, torch.Tensor):
```

Comment:
Does this really happen? I don't expect that.

Reply:
![image](https://user-images.githubusercontent.com/15828974/79860349-25ee8600-83f0-11ea-8552-3e4c04a1bf97.png)

You can see the type of inputs, which im printing.

Reply:
I still don't see how we get torch.Tensor as input? Inputs should always be relay value. From Torch tensor, we should get the numpy array, and wrap relay.const on it. Having torch tensor in op inputs is a logic bug.

Reply:
![image](https://user-images.githubusercontent.com/15828974/79861643-5800e780-83f2-11ea-8bff-f9d76382d2c3.png)

Actually i never saw numpy array as input, see above image, here the second input is `tensor([[0, 0],
        [1, 0]])` which is torch.Tensor

Reply:
ok this is because of this line
https://github.com/apache/incubator-tvm/blob/22db299b33f05570db2a5a406bdb37b57198a822/python/tvm/relay/frontend/pytorch.py#L1832

Can you apply `_wrap_const` on it? See how badly it breaks the rest of test cases. Returning a raw torch tensor is not a good idea anyway.

Reply:
ok. i will fix this and do the change accordingly. thanks.